pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was it necessary to create a parent pom, or is that just to test/show that multi-module projects now work?

I think it is slightly confusing to have the parent pom here, since it's only for the example/test projects and not the rest of akka-grpc, and the projects are intended to be fairly standalone anyway. I think I'd prefer the separate projects - those also still work, I guess?

Reply:
Yes, it's mostly for testing, I can remove it! 👍

Reply:
removed